mapper/tedge_mapper/src/c8y_mapper.rs

```diff
@@ -1,29 +1,44 @@
+use crate::c8y_converter::CumulocityConverter;
+use crate::component::TEdgeComponent;
 use crate::mapper::*;
-use mqtt_client::Topic;
+use async_trait::async_trait;
+use tedge_config::TEdgeConfig;
+use tracing::{debug_span, Instrument};
 
-#[derive(Debug)]
-pub struct CumulocityMapperConfig {
-    pub in_topic: Topic,
-    pub out_topic: Topic,
-    pub errors_topic: Topic,
-}
+const CUMULOCITY_MAPPER_NAME: &str = "tedge-mapper-c8y";
```

Comment:
How about moving it to the `TEdgeComponent` trait as all components will have some names anyway.

Reply:
I tried that, but it doesn't work because the instrument stuff (`debug_span!`) needs a const string.

Reply:
The mappers are in a library. Not sure what value putting them in separate crates adds.

My plan is to migrate collectd_mapper in a separate PR (still part of CIT-380). I just want to get the structure right first.

Reply:
> I tried that, but it doesn't work because the instrument stuff (`debug_span!`) needs a const string.

I remember someone else having had issue with that as well, can I suggest we may want to remove the `instrument` part which would then let us not use `span` and therefore move have the name as part of the trait?

Reply:
Well I can always keep the `const` and keep the `span` and just return the const from the function in the trait. But at the moment there would then not really be anything to call that method on the trait. The only other place we use the name is in `create_mapper` but that's an internal implementation detail of the trait implementation.